lib/avro_ex/decode.ex

```diff
@@ -169,8 +168,10 @@ defmodule AvroEx.Decode do
   def do_decode(%Array{items: item_schema}, %Context{} = context, data) when is_binary(data) do
     {count, buffer} = do_decode(%Primitive{type: :long}, context, data)
 
+    times = if count > 0, do: 1..count, else: []
```

Comment:
I do not love this at all. I would like something better. Ideas?

Reply:
I can't think of a better solution. Extracting the reduction into a helper function is not very clean, either. I'm okay with this. The `times` name doesn't quite fit the data structures, but it fits the usage.